src/main/java/org/folio/circulation/domain/Note.java

```diff
@@ -0,0 +1,44 @@
+package org.folio.circulation.domain;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getArrayProperty;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class Note {
+  private final JsonObject representation;
+
+  public Note(JsonObject representation) {
+    this.representation = representation;
+  }
+
+  public static Note from(JsonObject representation) {
+    return new Note(representation);
+  }
+
+  String getId() {
+    return getProperty(representation, "id");
+  }
+
+  String getTypeId() {
+    return getProperty(representation, "typeId");
+  }
+
+  String getDomain() {
+    return getProperty(representation, "domain");
+  }
+
+  String getTitleString() {
+    return getProperty(representation, "title");
+  }
+
+  String getTitleContent() {
+    return getProperty(representation, "content");
+  }
+
+  JsonArray getLinks() {
+    return getArrayProperty(this.representation, "links");
+  }
+
+}
```

Comment:
@dcrossleyau Does FOLIO have a standard for whether there should be an empty line at the end of a file?

Reply:
Yes, but not an explicit statement. And not an "empty line" as such (although many developers do want that) but at least a "final newline" (i.e. end-of-line character). Some text editors seem to omit that, thereby causing unnecessary diffs, and in some cases causing fragile systems.

The Guidelines
https://dev.folio.org/guidelines/contributing/#coding-style
and
https://dev.folio.org/guidelines/create-new-repo/#add-initial-files
do encourage the use of a "`.editorconfig`" file. That has this setting: `insert_final_newline = true`